plugins/permission/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@backstage/plugin-permission",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "homepage": "https://backstage.io",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/backstage/backstage",
+    "directory": "plugins/permission"
+  },
+  "keywords": [
+    "backstage"
+  ],
+  "scripts": {
+    "build": "backstage-cli plugin:build",
+    "start": "backstage-cli plugin:serve",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "dependencies": {
+    "@backstage/core-plugin-api": "^0.2.0",
+    "@backstage/plugin-permission-common": "^0.1.0",
+    "cross-fetch": "^3.0.6",
+    "react": "^16.13.1",
+    "react-router": "6.0.0-beta.0",
+    "react-use": "^17.2.4"
+  },
+  "devDependencies": {
+    "@backstage/cli": "^0.9.0",
+    "@backstage/test-utils": "^0.1.22",
```

Comment:
`@backstage/test-utils` imports this package (in order to use the `PermissionApi` type). But I think this circular dependency is okay because this is a dev dependency. Please correct me if I'm wrong!